### PR TITLE
[LongTail] Comment out invalid rules, closes #4216

### LINKED ISSUE #4216: Videos won't load on TeamFourStar.com
http://teamfourstar.com/video/dragonball-z-abridged-episode-52-flashpoint/

An error message comes up on videos unless the rule Longtail (partial) is disabled.

### PR DESCRIPTION
**Pull request by funkydude**
Simply a quick review of all the domains in this rule to fix https://github.com/EFForg/https-everywhere/issues/4216 before next release.

### CODE REVIEW

**Comment by numismatika** [comment]
i recently saw 
`<target host="jwpltx.com" />` 
without any subdomains being used. Haven't found out yet if it breaks stuff.

**Comment by funkydude** [comment]
@numismatika 
>You should also get rid of the merge commit.

If I knew how to do that or why it keeps getting added, I would.

**Comment by funkydude** [comment]
Now also resolves https://github.com/EFForg/https-everywhere/issues/4259

The review process really needs accelerated.

**Comment by numismatika** [comment]
maybe @J0WI or @fuglede might be able to help out before more issues get added to this pull request

File: src/chrome/content/rules/LongTail.xml [hunk 1]
```diff
@@ -36,23 +36,25 @@
...
 	<target host="securelp.longtailvideo.com" />
-	<target host="ltvimg.com" />
-	<target host="s0-www.ltvimg.com" />
-	<target host="s1-www.ltvimg.com" />
+	<!-- <target host="ltvimg.com" /> -->
+	<!-- <target host="s0-www.ltvimg.com" /> -->
+	<!-- <target host="s1-www.ltvimg.com" /> -->
 	<target host="jwplayer.com" />
 	<target host="account.jwplayer.com" />
 	<target host="www.jwplayer.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/LongTail.xml]: WFM:
support.jwplayer.com
checkout.jwplayer.com
developer.jwplayer.com
dashboard.jwplayer.com

Timeout:
jw7.jwplayer.com
demo.jwplayer.com
sdk.jwplayer.com

Mismatch:
insights.jwplayer.com
status.jwplayer.com

File: src/chrome/content/rules/LongTail.xml [hunk 2]
```diff
@@ -36,23 +36,25 @@
...
+	<target host="jwpltx.com" />
 	<target host="content.longtailvideo.com" />
 	<target host="dashboard.longtailvideo.com" />
 	<target host="l.longtailvideo.com" />
 	<target host="lp.longtailvideo.com" />
-	<target host="player.longtailvideo.com" />
+	<!-- <target host="player.longtailvideo.com" /> -->
 	<target host="plugins.longtailvideo.com" />
 	<target host="secure.longtailvideo.com" />
 	<target host="securelp.longtailvideo.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/LongTail.xml]: WFM:
longtailvideo.com
www.longtailvideo.com
account.longtailvideo.com
playertest.longtailvideo.com

**Comment by J0WI** [comment]
Please also update the comments above the rule.

**Comment by numismatika** [comment]
ssl.p.jwpcdn.com has a valid cert, does it break sites or why did you comment it out?

**Comment by funkydude** [comment]
@numismatika Maybe you're using old DNS servers, but that is not a valid domain.

**Comment by J0WI** [comment]
See ee105abf

**Comment by funkydude** [comment]
@J0WI done

**Comment by J0WI** [comment]
Could you readd the breakage example for ssl.p.jwpcdn.com?
And why did you changed the securecookie host?

**Comment by funkydude** [comment]
I don't know what diff you're looking at. I didn't remove any form of "breakage example" so there's nothing to "readd". I didn't change the cookie host, it's still the same, I removed the pointless subdomain restriction by simplifying the matching rule.

**Comment by funkydude** [comment]
Looks like 5.1.4 got merged and this is sitting stale. I'm just going to stick to my own contributions in future instead of trying to help close tickets, this project has serious management issues.

Consider this PR done, do what you want with it.

**Comment by J0WI** [comment]
I'm sorry that this didn't hit the 5.1.4 release, but there is still the uncommented change on securecookie that removes the wildcard.

**Comment by funkydude** [comment]
This issue doesn't affect me as I don't use those sites, apologizing to me is moot. I merely donated my time to help resolve what seems like a wide ranging issue.

You don't seem to understand how find functionality works. The current version will find and match more than the previous one. However, you're free to make any modifications you want, I won't be.

**Comment by J0WI** [comment]
Okay, thanks for your work anyway. I'll merge this and revert the securecookie change in a follow-up commit.

**Pull request merged**